docusaurus.config.ts

```diff
@@ -72,8 +72,19 @@ const config: Config = {
         sitemap: {
           changefreq: EnumChangefreq.WEEKLY,
           priority: 0.5,
+          lastmod: 'date',
+          // Ignore all versions except the latest one
```

Comment:
I think this lastmod is already the default:
![image](https://github.com/user-attachments/assets/46af0716-7a9c-4b0b-8ef3-bfd7429d89dd)

Also remove the comment as it already exists on line 83